apps/frontend/craco.config.js

```diff
@@ -1,4 +1,14 @@
+const CracoEnvPlugin = require('craco-plugin-env');
```

Comment:
CRA loads env out of the box so you dont need plugins:

https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used

https://create-react-app.dev/docs/adding-custom-environment-variables/#referencing-environment-variables-in-the-html